sdk/schemaregistry/schema-registry-avro/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2020-09-08)
```

Comment:
Sorry to ask, is this the correct release date?

Reply:
We're still hoping to push the package today, but if that doesn't happen, talking @ramya-rao-a, it didn't seem like a problem if this is a day or two earlier  than the real date.